app/views/comments/_single_comment.html.erb

```diff
@@ -60,7 +60,9 @@
       <% end %><!-- end caching -->
 
       <%= render "comments/comment_actions", comment: single_comment %>
-
+      <% if logged_in_as_admin?  %>
```

Comment:
We use "No address recorded" for the IP address in the work meta -- we should use the same phrasing here. Also, this should come above`<%= render "comments/comment_actions", comment: single_comment %>`, and have some styling applied to better differentiate it from the comment text. I'd suggest adding the following to 24-role-admin.css, directly above `/* TABLES AND LISTS */`:

```
.comment .ip {
  margin: 1.929em 0.643em 0.643em;
  text-align: right;
}
```

(If you could remove the `#footer .beta` bit on lines 12-14 while you were in that file, I'd be much obliged. We took the beta stripe out of the footer a long time ago and I apparently forgot to update this file.)

There are also a few spacing issues: 
- There's an extra space between `logged_in_as_admin?` and ` %>`
- The `<p>` is indented two spaces too many
- There's an extra space between the text and the closing `</p>` tag

We should probably add the same IP address information in comments/_single_comment_abbreviated.html.erb as well so Abuse can see it from the /comments index.